wp-content/themes/pub/wporg-learn-2020/archive-course.php

```diff
@@ -20,10 +20,12 @@
 
 <main id="main" class="site-main">
 	<section>
-		<div class="section-heading section-heading--with-space row align-middle between">
-			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+		<div class="section-heading section-heading--with-space row align-middle between gutters">
+			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
 			<?php if ( is_user_logged_in() ) : ?>
-				<a class="section-heading_link" href="/my-courses/"><?php esc_html_e( 'My Courses', 'wporg-learn' ); ?></a>
+				<div class="col-4 row section-heading_links">
+					<a class="section-heading_link" href="/my-courses/"><?php esc_html_e( 'My Courses', 'wporg-learn' ); ?></a>
+				</div>
```

Comment:
This pattern ensures there is space between the elements both on desktop and smaller devices, and keeps the title left aligned, and any right elements right aligned within their column.